src/program/lwaftr/bench/README

```diff
@@ -12,16 +12,19 @@ Usage: bench CONF IPV4-IN.PCAP IPV6-IN.PCAP
 
                                Time (s),Decap. MPPS,Decap. Gbps,Encap. MPPS,Encap. Gbps
   -b FILENAME, --bench-file FILENAME
-                              The file or path name to which benchmark data is
-                              written. A simple filename or relative pathname
-                              will be based on the current directory. Default
-                              is "bench.csv".
+                             The file or path name to which benchmark data is
+                             written. A simple filename or relative pathname
+                             will be based on the current directory. Default
+                             is "bench.csv".
   -D DURATION, --duration DURATION
                              Duration in seconds.
   -n NAME, --name NAME       Sets the name for this program, which will be used
                              as the identifier. This must be unique amongst
                              other snabb processes.
   --cpu    <cpu>             Bind lwAFTR bench to the given CPU
+  --reconfigurable           Allow run-time configuration query and update.
+                               This runs as two processes, a leader and a
```

Comment:
AFAIU these descriptions are intended to be aligned.  If that is so, would you mind aligning lines 26 and 27?  Tx.